week2/homework/src/remove.js

```diff
@@ -0,0 +1,29 @@
+const fs = require('fs');
+
+function remove(toDoItem) {
+  fs.readFile('./toDo.json', 'utf8', (error, data) => {
```

Comment:
What would happen if you would like to change the location where the file is stored?